OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -115,7 +115,14 @@ void DisguiseAs(Actor self, Actor target)
 				var tooltip = target.TraitsImplementing<IToolTip>().FirstOrDefault();
 				AsTooltipInfo = tooltip.TooltipInfo;
 				AsPlayer = tooltip.Owner;
```

Comment:
Why? :) As far as I can see, that's what you did.
But only in one branch of the if-block, instead of removing this line and having each branch set the correct `AsPlayer`.